apis/contour/v1beta1/ingressroute.go

```diff
@@ -77,6 +77,14 @@ type Delegate struct {
 	Namespace string `json:"namespace"`
 }
 
+// Status reports the current state of the IngressRoute
+type Status struct {
+	CurrentStatus   string      `json:"currentStatus"`
+	Description     string      `json:"description"`
+	LastProcessTime metav1.Time `json:"lastProcessTime"`
+	Paths           []string    `json:"paths"`
```

Comment:
I thought that we weren't going to include this at the moment because the decision for beta.1 was status would be all or nothing -- everything in the document worked, or nothing in the document worked; and we'll capture that weakly in the `description:` key